web/src/components/storage/BootSelection.tsx

```diff
@@ -37,12 +38,16 @@ import {
 import textStyles from "@patternfly/react-styles/css/utilities/Text/text";
 import { sprintf } from "sprintf-js";
 import { _ } from "~/i18n";
+import { configModelMethods } from "~/model/storage";
+import type { configModel } from "~/model/storage";
 import type { storage } from "~/model/system";
-import type { Model } from "~/storage/model";
 
-const filteredCandidates = (candidates: storage.Device[], model: Model): storage.Device[] => {
+const filteredCandidates = (
+  candidates: storage.Device[],
+  configModel: configModel.Config,
```

Comment:
this is a bit tricky to parse...we have variable `configModel` that has type `configModel.Config`...it looks a bit strange for me. I would probably just call it `config` instead of `configModel` especialy when the Type is config

Reply:
and if I see correctly it is only usage of configModel type here...so is possible to just import Config part of configModel?

Reply:
I hope now it is clearer because the group type is called `ConfigModel` instead of `configModel`.